client/packages/requisitions/src/RequestRequisition/DetailView/Toolbar/ToolbarActions.tsx

```diff
@@ -54,6 +54,9 @@ export const ToolbarActions = () => {
                   onConfirm: () => update({ minMonthsOfStock: option.value }),
                 })
               }
+              getOptionDisabled={(option) => {
```

Comment:
nice! that's a very neat solution! 😁 

( just noticed, it can be simplified a bit.. )

```suggestion
              getOptionDisabled={option => option.value > maxMonthsOfStock}
```

Reply:
oh! and just to be a little nit picky.. you can set the max to be less than the min 🙄 

<img width="352" alt="image" src="https://user-images.githubusercontent.com/9192912/231311106-82986a76-286f-4261-845d-7eaed10cb631.png">

would you mind doing the same sort of thing for the max drop down too?

I do wonder if we should have a check in the API as well.. but I'm not fussed about that.

Reply:
Done (: